### ISSUE TITLE
Use .gitkeep instead of git ignore

### ISSUE THREAD

**Issue by MahdiBaghbani** [pondersource/CYB#13]
discussed in #12

**Comment by michielbdejong** [comment]
I just read about it as well, it's explained well in https://stackoverflow.com/questions/7229885/what-are-the-differences-between-gitignore-and-gitkeep

**Comment by MahdiBaghbani** [comment]
I think that we really don't want those files at all, /bootstrap and /storage are created during Laravel app runtime, for caching and other stuff.
I was kinda wrong to add empty .gitignores, should we add storage and bootstrap to our main .gitignore in the root of project?

**Comment by michielbdejong** [comment]
Yes! Something like https://github.com/github/gitignore/blob/main/Laravel.gitignore

**Comment by michielbdejong** [comment]
Good point, that's probably how Laravel development on git is supposed to work.
